getting-started/quick-start.md

```diff
@@ -0,0 +1,110 @@
+# Getting Started with Amber
+This getting started guide will help you get a full stack web application running in just a few minutes by leveraging amber’s code generators.
+
+After we have a working application, then we will walk through the code that is making it all happen and code some enhancements along the way.
+
+Let’s get started!
+
+## Install dependencies
+If you already have `crystal`, `amber`, and a database installed, you can skip this step.
+
+### Install `crystal`
+
+#### OS X with homebrew
+ ```
+$ brew install crystal-lang
+```
+#### Ubuntu or Debian
```

Comment:
What about adding `yaourt -S amber` for ArchLinux? 😅

Reply:
Is there a way to make that auto install crystal as a dependency as well?

Reply:
crysta is already an amber dependency on amber-aur, see https://github.com/amberframework/amber-aur/blob/master/PKGBUILD#L9